source/slang/hlsl.meta.slang

```diff
@@ -2296,10 +2296,14 @@ ${{{{
         {
             auto componentName = componentNames[componentId];
             auto glslComponent = (isCmp ? "" :glslComponentNames[componentId == 0 ? 0 : componentId - 1]);
-    }}}}
+    
+            for (bool isStatus : { false, true })
+            {
+                const char* statusDecl = isStatus ? ", uint status" : "";
```

Comment:
status is an `out` [parameter](https://learn.microsoft.com/en-us/windows/win32/direct3dhlsl/t2d-gather-s-float-int-uint-)
```suggestion
                const char* statusDecl = isStatus ? ", out uint status" : "";
```